TestBots/Bridge/SAYC/1NT Openings.pbn

```diff
@@ -1,43 +1,38 @@
-[Event "1NT Opening - All Pass (modified)"]
```

Comment:
This diff is just uncommenting the disabled tests (and removing the modified one I added when enabling PBN support).